test/benchmark/bm_ursa_ed25519.cpp

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "ursa_crypto.h"
+
+#include <cstdlib>
+#include <vector>
+
+#include <benchmark/benchmark.h>
+
+auto ConstructRandomVector(int64_t size) {
+  std::vector<uint8_t> v;
+  v.reserve(static_cast<int>(size));
+  for (int i = 0; i < size; ++i) {
+    v.push_back(static_cast<uint8_t>(std::rand() % size));
```

Comment:
on line 17 in bm_iroha_ed...cpp you have been using "unsigned char." Probably it worth to use the same type alias in both cases for consistency.